weareinreach/inreach-catalog
# Adds new languages to the property map

## Description

Adds new languages to the property map
- Afrikaans
- Azerbaijani
- Catalan
- Czech
- Dutch
- English
- Flaams
- Ga
- Guarani
- Ixil
- Kinyarwanda
- Ladino
- Lithuanian
- Malagasy
- Malinke
- MSL (Mexican Sign Language)
- Oromo
- Pulaar
- Rohingya
- Shona
- Swedish
- Tajiki
- Tamazight
- Yiddish
- Yoruba

- Asana ticket: https://app.asana.com/0/1132189118126148/1191733863456459/f

## PR Checklist


## How to Test

1. Navigate to the Control Panel
2. Log in
3. Select an organization to view
4. Select a service to view
5. Navigate to the Properties tab
6. Click "Edit Properties"
7. Scroll down the properties dialog and observe that corresponding language properties for the aforementioned languages are now available for selection
8. Select any number of the new languages and click "Save"
9. Click "View on Catalog" to see the updated service on the AC catalog
10. Observe that all the language properties selected are being displayed on the service page
